src/schema/v2/artwork/index.ts

```diff
@@ -110,7 +110,7 @@ const ArtworkPriceInsightsType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
       description: 'The annual value of the work sold "in USD "',
       resolve: ({ annualValueSoldCents }) => {
-        return `$${formatLargeNumber(annualValueSoldCents)}`
+        return `$${formatLargeNumber(annualValueSoldCents / 100)}` // dividing by 100 because we don't need the value in cents
```

Comment:
We should avoid doing currency math in multiple, local, view-oriented places. Better touse the money helpers that are (I think) already available in this project, and avoid currency differences, rounding, floating-point precision issues, etc.

Reply:
+1 to this! Baking in `/ 100` makes it much harder to support different currencies in the future.

Reply:
@dariakoko let's fix this and use the money helper then.